src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AddingNewEventArgs.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+
+
+
```

Comment:
This is a by-product of removing the old style copyright headers. I will see if the CodeFormatter tool can fix this. If it cannot, I am going to leave it as-is.